.travis.yml

```diff
@@ -3,12 +3,12 @@ script:
   - cd $TRAVIS_BUILD_DIR
   - cargo build
   - cargo test
-  - cargo build --features serde
-  - cargo test --features serde
+  - cargo build --all-features
+  - cargo test --all-features
```

Comment:
It might be good sometimes to also test if various combinations of features being turned on/off work as well - but if you feel confident that the features are independent enough to just test no features and all of them, I'm good with that :wink:

Reply:
Yeah, the only two features we have now are orthogonal, so it's fine. Just makes testing quicker.